python/ql/test/experimental/query-tests/Security/CWE-943/NoSQLInjectio.qlref

```diff
@@ -0,0 +1 @@
+experimental/Security/CWE-943/RegexInjection.ql
```

Comment:
This file is misspelled. Will fix it after we get initial feedback.

Reply:
It also currently references `RegexInjection.ql` which is probably not what you want :)

Reply:
Resolved.

Fixed the file name and referenced it to `experimental/Security/CWE-943/NoSQLInjection.ql`.